src/formatting_stack/linters/eastwood/impl.clj

```diff
@@ -1,9 +1,18 @@
 (ns formatting-stack.linters.eastwood.impl
   (:require
+   [clojure.string :as string]
    [eastwood.reporting-callbacks :as reporting-callbacks]
    [formatting-stack.protocols.spec :as protocols.spec]
    [nedap.speced.def :as speced]))
 
+(speced/defn ^boolean? wrong-pre-post-false-positives
+  "Removes false positives for dynamic vars https://git.io/fhQTx"
+  [{{{[_fn* [_arglist [_assert v]]] :form} :ast} :wrong-pre-post}]
+  (let [varname (-> v str (string/split #"\/") last)]
```

Comment:
Did you try `(name v)` instead?

Reply:
I did nort, but  `v` is not guaranteed to be a symbol.

we could just add an explicit check for `symbol?` though, in our case we only want to remove if it's a sym.

will add test for `"*not-ignored*"` as well